### PR TITLE
Updated documentation, added screenshots for rc3

### PR DESCRIPTION
**Pull request by tonya1**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23discussion_r172720488%22%2C%20%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23issuecomment-371043734%22%2C%20%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23issuecomment-371045971%22%2C%20%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23discussion_r173310302%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23issuecomment-371043734%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20is%20there%20a%20Future%20enhancements%20in%20the%20installation%20steps.%20It%20should%20just%20be%20Next%20step%3A%20Using%20GridAPPS-D%20%22%2C%20%22created_at%22%3A%20%222018-03-07T07%3A03%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/6538501%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/poorva1209%22%7D%7D%2C%20%7B%22body%22%3A%20%22There%20are%20a%20lot%20of%20changes%20needed%20in%20the%20Using%20GridAPPS-D%20section%20but%20we%20can%20merge%20this%20pull%20request%20and%20then%20work%20on%20those%20changes%20next.%20Good%20job%20Tonya.%20%22%2C%20%22created_at%22%3A%20%222018-03-07T07%3A15%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/6538501%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/poorva1209%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20eb82795693c2c86b32b0df41942411b6d03938d1%20docs/source/using_gridappsd/rc1_overview.rst%2058%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446%23discussion_r172720488%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Should%20these%20images%20be%20changed%20to%20reflect%20the%20rc3%22%2C%20%22created_at%22%3A%20%222018-03-07T01%3A54%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/3979063%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/craig8%22%7D%7D%2C%20%7B%22body%22%3A%20%22Created%20a%20new%20pull%20request%20with%20updated%20images%20and%20Future%20Enhancements%20removed.%5Cr%5Cn%5Cr%5Cnhttps%3A//github.com/GRIDAPPSD/gridappsd-docs/pull/1%22%2C%20%22created_at%22%3A%20%222018-03-08T22%3A17%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/15236222%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tonya1%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20docs/source/using_gridappsd/rc1_overview.rst%3AL77-84%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull eb82795693c2c86b32b0df41942411b6d03938d1 docs/source/using_gridappsd/rc1_overview.rst 58'></a> &nbsp;<b><a href='https://github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446#discussion_r172720488'>File: docs/source/using_gridappsd/rc1_overview.rst:L77-84</a></b>
- <a href='https://github.com/craig8'></a> Should these images be changed to reflect the rc3
- <a href='https://github.com/tonya1'></a> Created a new pull request with updated images and Future Enhancements removed.
https://github.com/GRIDAPPSD/gridappsd-docs/pull/1
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446#issuecomment-371043734'>General Comment</a></b>
- <a href='https://github.com/poorva1209'></a> Why is there a Future enhancements in the installation steps. It should just be Next step: Using GridAPPS-D
- <a href='https://github.com/poorva1209'></a> There are a lot of changes needed in the Using GridAPPS-D section but we can merge this pull request and then work on those changes next. Good job Tonya.

<a href='https://www.codereviewhub.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446?approve=1'></a>&nbsp;<a href='https://github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by craig8** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/GRIDAPPSD/GOSS-GridAPPS-D/pull/446'>Refresh</a>.<a href='#crh-update-none'></a>

File: docs/source/using_gridappsd/rc1_overview.rst [hunk 1]
```diff
@@ -51,7 +77,7 @@ For an orderly shutdown of the demonstration:
 
 |rc1_overview_image0|
```
Comment (craig8) [inline docs/source/using_gridappsd/rc1_overview.rst]: Should these images be changed to reflect the rc3

**Comment by poorva1209** [comment]
Why is there a Future enhancements in the installation steps. It should just be Next step: Using GridAPPS-D

**Comment by poorva1209** [comment]
There are a lot of changes needed in the Using GridAPPS-D section but we can merge this pull request and then work on those changes next. Good job Tonya.

File: docs/source/using_gridappsd/rc1_overview.rst (hunk 1 above)
Comment (tonya1) [reply docs/source/using_gridappsd/rc1_overview.rst]: Created a new pull request with updated images and Future Enhancements removed.

https://github.com/GRIDAPPSD/gridappsd-docs/pull/1

**Pull request closed without merging**